core/templates/pages/exploration-player-page/learner-experience/conversation-skin.directive.ts

```diff
@@ -46,11 +47,13 @@ require('domain/exploration/editable-exploration-backend-api.service.ts');
 require('domain/exploration/read-only-exploration-backend-api.service.ts');
 require('domain/question/pretest-question-backend-api.service.ts');
 require('domain/skill/ConceptCardObjectFactory.ts');
-require('domain/state_card/StateCardObjectFactory.ts');
+require('domain/state_card/state-card.model.ts');
```

Comment:
Do we still need this?

Reply:
No, thanks for pointing out.
Updated the commit.